samples-generator/lib/createBatchTableHierarchy.js

```diff
@@ -29,13 +31,15 @@ var whiteOpaqueMaterial = new Material({
 });
 
 /**
- * Create a tileset that uses a batch table hierarchy.
+ * Create a tileset that uses a batch table hierarchy,
+ * by default using the 3DTILES_batch_table_hierarchy extension.
  *
  * @param {Object} options An object with the following properties:
  * @param {String} options.directory Directory in which to save the tileset.
  * @param {Boolean} [options.batchTableBinary=false] Create a batch table binary for the b3dm tile.
  * @param {Boolean} [options.noParents=false] Don't set any instance parents.
  * @param {Boolean} [options.multipleParents=false] Set multiple parents to some instances.
+ * @param {Boolean} [options.legacy=false] Generate the batch table hierarchy as part of the base tileset JSON file, now deprecated.
```

Comment:
Tweak the description a bit: `Generate the batch table hierarchy as part of the base batch table, now deprecated.`